apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -149,13 +153,20 @@ export default connect(
           );
         } else if (this.props.isAttached) {
           try {
-            const result = this.props.evalInCurrentScope(input);
-            this.appendLog('< ' + String(result));
+            let result = this.props.evalInCurrentScope(
+              input[0] === '{' && input[input.length - 1] === '}'
```

Comment:
can you explain why this is desired functionality?

Reply:
Of course! Currently the debug console on prod is not returning lists and objects correctly like the javascript console would return lists and objects. For example, if a student's input was ["a", "b"] then it would return "a,b" as the output to the student and if input was {foo: "bar"} then only the value "bar" would be returned. 

The output of the list is correctly displayed by removing String() of String(result) in line 153 on the left side. But objects need additional work so line 157 is checking that the input is a hash and then wrapping it with () before the evalInCurrestScope because that ensured the output is always returning correctly:

<img width="341" alt="Screen Shot 2019-05-17 at 12 58 13 PM" src="https://user-images.githubusercontent.com/17921445/57953054-e34d3100-78a3-11e9-9e0a-06067eeb1ccb.png">